.github/workflows/test.yml

```
@@ -19,7 +19,8 @@ jobs:
         with:
           check_filenames: true
           ignore_words_list: atleast,ninjs,simpy,proovread,namd,precice
-          exclude_file: docs/js/asciinema-player-2.6.1.js
+          # filter out version-specific/* for now, until we fix all issues that mdl trips over
+          exclude_file: 'docs/js/asciinema-player-2.6.1.js|docs/version-specific/*'
```

```suggestion
          skip: './docs/js/asciinema-player-2.6.1.js, ./docs/version-specific'
```

Exclude file is the wrong option.